dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -457,6 +457,14 @@ public void PaneClosedWhenAutoAndNarrow()
                     Log.Comment("Checking that a NavigationView with displaymode set to 'Auto' and a narrow width does not display pane");
                     Wait.ForIdle();
                     Verify.IsTrue(isPaneOpenCheckBox.ToggleState == ToggleState.Off);
+
+                    isPaneOpenCheckBox = new CheckBox(FindElement.ById("IsLeftMinimalPaneOpenCheckBox"));
```

Comment:
Don't re-use a variable for a new control. It is confusing. Name this variable something like 'isLeftMinimalPaneOpenCheckBox'